fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -403,8 +403,8 @@ private void addCompositeMigrationStepsV0009(List<IDatabaseStatement> statements
         // Simplified composite value support. Eliminate the composites table and instead
         // use a new composite_id column to correlate parameters involved in a composite relationship
         // drop PK constraint; remove ROW_ID column after data migration
-        statements.add(new DropPrimaryKey(schemaName, tableName));
-        statements.add(new DropColumn(schemaName, tableName, ROW_ID));
+        statements.add(new DropPrimaryKey(schemaName, tableName, true));
+        statements.add(new DropColumn(schemaName, tableName, true, ROW_ID));
```

Comment:
so this ignores any errors?

Reply:
Yes. Ideally we'd make it idempotent for all database types, but to do this properly means we should refactor Migration so that it's based on commands and adapters instead of raw statements. I'll create an issue.

Reply:
issue #2031